substrafl_examples/go_further/README.rst

```diff
@@ -0,0 +1,2 @@
+Example to go further
+^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The titles feel huge. Let's try to make it a bit smaller:
![image](https://user-images.githubusercontent.com/5354301/200580353-a9c3ec21-77c7-4ddb-91fa-b557237bf34c.png)

Reply:
I don't think it's possible...

Reply:
Ok not a big deal